src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -159,7 +159,8 @@ def _validate_ip_configuration(namespace, lab_vnet=None):
     elif namespace.ip_configuration is None:
         # case 5: lab virtual network was selected from user's option / formula default then use it for look-up
         if lab_vnet:
-            if lab_vnet.subnet_overrides and lab_vnet.subnet_overrides[0].shared_public_ip_address_configuration:
+            if lab_vnet.subnet_overrides and lab_vnet.subnet_overrides[0].shared_public_ip_address_configuration and \
+                    lab_vnet.subnet_overrides[0].shared_public_ip_address_configuration.inbound_nat_rules:
```

Comment:
Without a comments on it, I doubt the next reader can figure out what's going on quickly.